### ISSUE TITLE
Merge check_report_used_space, check_report_free_space, check_free_space probes

### ISSUE THREAD

**Issue by ericvaandering** [dmwm/CMSRucio#672]
**Task description**
These three probes should be merged into one. And the minfree space which ops is generating another way should be added. 

Probe should be modernized with Prometheus pusher, etc as needed. 

**Related issues**
- [ ] Please link your PRs on the right sidebar
- [ ] Update dashboards using these probes (please note, old data may continue to flow from an old cluster still running probes)

**Technical Implementation**
Describe the technical implementation needed, and what are the expected problems/questions 

**Action items**
Sub-Items to be tracked
- []

**Comment by ericvaandering** [comment]
Just so you are aware, the code we actually run is in https://github.com/dmwm/CMSRucio/blob/master/docker/rucio-probes/Dockerfile and I see that at least one of your probes is changed in that PR, so make sure you start from there, not what is in the Rucio repository. I have been trying for months to get some of these changes pushed in.

**Comment by ericvaandering** [comment]
And here's a PR for doing something similar for three other probes: https://github.com/rucio/probes/pull/122

**Comment by Panos512** [comment]
thanks @ericvaandering ! I took your code and built it on top of it by combining the three probes and adding the `minfreespace` calculation.

I have it here https://github.com/Panos512/probes/commit/9c9801e77ec5b4848fbbaa4b9d2ba42bb6654260

I've also commented out the gauge lines and run the code in the probes pod, imitating a dry run. The results are as expected.

The only thing is that `rucio_free_space` is always 0.

This is calculated with the following logic:

```python3
if usage['source'] == 'rucio':
                        rucio_free_space = int(usage['total']) - int(usage['used'])
```
`total` and `used` are always equal. 
I think `rucio` is the total space (unavailable + expired + locked) which means that total and used should always be equal.
Do we want to monitor this diff to see if everything is working as expected? Could total and used diverge in case something is going wrong?

I will anyways prepare a MR for the probes module in core rucio, hardcode my new file in the dockerfile for the probes image you linked above and change the integration cluster configuration in flux to deploy it there.

@haozturk yesterday observed that probes send the metrics in prometheus together with the instance name, this means that it would be safe to test the probes in integration since they wouldn't interfere with the production data
